src/Voryx/ThruwayBundle/WampKernel.php

```diff
@@ -228,14 +228,21 @@ public function handleRPC($args, $argsKw, $details, MappingInterface $mapping)
      */
     protected function createSubscribe(MappingInterface $mapping)
     {
-        $topic = $mapping->getAnnotation()->getName();
+        /**
+         * @var Subscribe $annotation
+         */
+        $annotation = $mapping->getAnnotation();
+
+        $topic = $annotation->getName();
 
         $subscribeCallback = function ($args, $argsKw, $details) use ($mapping) {
             $this->handleEvent($args, $argsKw, $details, $mapping);
         };
 
         //Subscribe to a topic
-        $this->session->subscribe($topic, $subscribeCallback);
+        $this->session->subscribe($topic, $subscribeCallback, [
+            'match' => $annotation->getMatch(),
+        ]);
```

Comment:
Would it be better to only send a match option when it is set in the annotation instead of having the default `exact` sent for every subscription?

Reply:
That's a good shout, that'll help us always use the correct default when the core library changes. Will deal with this in the morning!